README.md

```diff
@@ -380,6 +380,22 @@ To create a Messenger button, call the `createMessengerButton` method. Example:
     win.add(messengerButton);
 ```
 
+Deferred App links
+------------------
+
+Deferred deep linking allows you to send people to a custom view after they installed your app via the app store.
+
+You can simply call fetchDeferredAppLink on startup to open eventually incoming app links.
+
+```javascript
+var fb = require('facebook');
+fb.fetchDeferredAppLink(function(e) {
+    if (e.url) {
```

Comment:
Because we are checking the url-existence internally, we should just check `e.success` here like done in other API's.